layouts/css/layout/_sticky-footer.styl

```diff
@@ -13,7 +13,8 @@ body
 #main
     flex 1 0 auto
 
-header
+header,
+footer
```

Comment:
Prevent some browsers from letting this item shrink to smaller than its content's default minimum size.